apps/web/components/Forms/PoolForm.tsx

```diff
@@ -390,18 +393,19 @@ export function PoolForm({ token, communityAddr, chainId }: Props) {
             <h4 className="my-4 text-xl">Select pool settings</h4>
             <div className="flex gap-8">
               {Object.entries(poolSettingValues).map(
-                ([key, { label, description }]) => (
```

Comment:
how about going with:

   ([key, { label, description }]) => (
                    <FormRadioButton
                      key={"poolSettings-"+key}
                      label={label}
                      description={description}
                      value={parseInt(key)}
                      checked={optionType === parseInt(key)}
                      onChange={handleOptionTypeChange}
                      registerKey="poolSettings"
                    />
                ),

Reply:
It really helps react identify in which element to attach the key, if you put it in the component it will try to determines it by himself and if the child component is rerendering with too much changes, he might get lost.

Reply:
The fragment is a more specific group reference